requirements.txt

```diff
@@ -1,49 +1,48 @@
-requests==2.31.0
-petl==1.6.8
+airtable-python-wrapper==0.13.0
+azure-storage-blob==12.3.2
 boto3>=1.17.98
 boxsdk==2.10.0
+braintree==4.0.0
+bs4==0.0.1
+censusgeocode==0.4.3.post1
 civis==1.16.0
-slackclient==1.3.0
-psycopg2-binary==2.9.3
-xmltodict==0.11.0
-gspread==3.7.0
-oauth2client==4.1.3
-google-auth==2.6.2
+dbt_redshift==1.4.0
+docutils<0.18,>=0.14
```

Comment:
@bmos do you remember why docutils is pinned like this? 

It's causing an issue bumping up sphinx in #1201 

https://app.circleci.com/pipelines/github/move-coop/parsons/6034/workflows/cba51139-5d87-4e55-9f68-ff53cf80afd9/jobs/7622?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary

Reply:
I think it's been that way since before my time

Reply:
git blame letting me down :( I will give it a try